src/_preflights/resets.js

```diff
@@ -0,0 +1,4 @@
+export const resets = {
+  layer: 'preflights',
+  getCSS: async () => (await fetch('https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css')).text(),
```

Comment:
We'd probably need to update that repo with changes from e.g. https://github.com/warp-ds/drive/pull/107/files

Reply:
Yes, absolutely. Just waiting for us to land on something here.

Reply:
Done